test/run-tests

```diff
@@ -578,6 +586,10 @@ def handle_task(gh, args, config, task, dry_run=False):
         )
     print()
 
+    if abort:
+        print("Fatal exception occured, aborting...")
+        sys.exit(1)
```

Comment:
Why don't you exit right away when the exception happens?

Please include a hint that the error happened because there's no space left on the device.

Reply:
The idea is to still set the status to "error" and upload the logs before exiting. Also the actual exception that mentions that no space is left on the device is printed in line 546.

Reply:
Ah, I overlooked that. Thanks!